docs/source/reference/series.rst

```diff
@@ -114,3 +114,27 @@ Serialization / IO / Conversion
    Series.to_json
    Series.to_csv
    Series.to_excel
+
+Datetime Methods
+----------------
+Methods accessible through `Series.dt`
+
+.. currentmodule:: databricks.koalas.datetime
+.. autosummary::
+   :toctree: api/
+
+   DatetimeMethods.date
```

Comment:
@HyukjinKwon can you figure out how to present the functions similar to pandas?

Reply:
Yup

Reply:
@HyukjinKwon Please cc me on the PR if you got it working. I banged my head against this for a while :(

Reply:
haha same to me too ;D

Reply:
k, I opened a PR - https://github.com/databricks/koalas/pull/404